libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1262,6 +1361,13 @@ export class RushConfiguration {
     return commonVersionsFilename;
   }
 
+  /**
+   * Does this project have subspaces
+   */
+  public get hasSubspaces(): boolean {
+    return this._subspaceProjectsCache.size > 0;
```

Comment:
Will this always be hydrated? The "Cache" part of the name make this sound like it may or may not be fully initialized.